docs/features/techdocs/how-to-guides.md

```diff
@@ -142,6 +142,78 @@ const AppRoutes = () => {
 };
 ```
 
+## How to customize the TechDocs reader page?
+
+### Custom Header
+
+This is done in your `app` package. By default, you might see something like
+this in your `App.tsx`:
+
+```tsx
+const AppRoutes = () => {
+  <FlatRoutes>
+    <Route
+      path="/docs/:namespace/:kind/:name/*"
+      element={<TechDocsReaderPage />}
+    />
+  </FlatRoutes>;
+};
+```
+
+But the `<TechDocsReaderPage />` component accepts children, which means that
+you can customize the page with any other Header component you would like to
+use.
+
+Most likely, you would want to create and maintain such a component in a new
+directory at `packages/app/src/components/techdocs`, and import and use it in
+`App.tsx`:
+
+```tsx
+<Route
+  path="/docs/:namespace/:kind/:name/*"
+  element={
+    <TechDocsReaderPage>
+      {({ techdocsMetadataValue, entityMetadataValue, entityId }) => (
+        <TechDocsCustomPageHeader
+          techDocsMetadata={techdocsMetadataValue}
+          entityMetadata={entityMetadataValue}
+          entityId={entityId}
+        />
+      )}
+    </TechDocsReaderPage>
+  }
+/>
+```
+
+And if your Header do not need access to specific metadata, you can just use
+something like:
+
+```tsx
+<Route
+  path="/docs/:namespace/:kind/:name/*"
+  element={
+    <TechDocsReaderPage>
+      {() => <Header type="documentation" title="Custom Header" />}
```

Comment:
**Suggestion:**

The component could accept `render function` or `react element` as children:

A render function:

```jsx
<TechDocsReaderPage>
  {({ techdocsMetadataValue, entityMetadataValue, entityId }) => (
    <TechDocsCustomPageHeader
      techDocsMetadata={techdocsMetadataValue}
      entityMetadata={entityMetadataValue}
      entityId={entityId}
    />
  )}
</TechDocsReaderPage>
```

Or just an element:

```jsx
<TechDocsReaderPage>
  <Header type="documentation" title="Custom Header" />
</TechDocsReaderPage>
```

Reply:
yes, that is a great suggestion, I'll update!